spec/isolator/adapters/background_jobs/active_job_spec.rb

```diff
@@ -17,5 +17,78 @@
     specify do
       expect { ActiveJobWorker.perform_later("test") }.to raise_error(Isolator::BackgroundJobError, /ActiveJobWorker.+test/)
     end
+
+    # TODO: replace with `ActiveJob.version >= Gem::Version.new("7.2")` once 7.2 is released
+    context "with enqueue_after_transaction_commit", if: ActiveJob::VERSION::MAJOR >= 7 && ActiveJob::VERSION::MINOR >= 2 do
```

Comment:
This is necessary since `ActiveJob.version >= Gem::Version.new("7.2")` will return false as Rails main is currently on `7.2.0-alpha`.

Reply:
Looks like we need to add a similar check to Action Mailer specs.

Maybe, it's worth moving the version check to the `spec_helper.rb` and using RSpec tags to exclude such tests?

Reply:
Is this what you were thinking: https://github.com/palkan/isolator/compare/7eeba75bffdd2312ebebedba0c86ae5af3abcdf0..1a0795725d80c031c1ba585b9fafc71e03351ce1